src/dictionaries.h

```diff
@@ -29,4 +29,20 @@ class DependentDictionary
         std::map<std::pair<std::string, std::string>, uint8_t> m_reverseDict;
 };
 
+class Dictionaries
+{
+    public:
+        Dictionaries();
```

Comment:
In C++11 and later (here we use C++17) if you have an emty constructor you can mark it as `default` to ask the compiler to generate it for you. So instead of having
```c++
Dictionaries();
```
in the header and
```c++
Dictionaries::Dictionaries()
{
}
```
in the cpp-file, you can write:
```c++
Dictionaries() = default;
```
in the header and nothing in the cpp-file.